install_config/aggregate_logging.adoc

```diff
@@ -440,8 +440,19 @@ endif::openshift-enterprise[]
 [[scaling-elasticsearch]]
 *Changing the Scale of Elasticsearch*
 
-To scale Elasticsearch up, create more deployments. The deployer provides
-a template to create Elasticsearch deployments:
+If you need to scale up the number of Elasticsearch instances your cluster uses
+it is not as simple as changing the number of replicas. This is due to the nature
+of Persistent volumes and how Elasticsearch is configured to store its data.
+
+Instead, you will need to create a Deployment Configuration for each replica.
+
+The Deployer created templates during installation with the Elasticsearch
+configurations provided to it: `logging-es-template` and `logging-es-ops-template`
+if the Deployer was run with `ENABLE_OPS_CLUSTER=true`.
+
+To create an additional Deployment Configuration, run the following command against
+the Elasticsearch cluster you'd like to scale up for (`logging-es-template` or
+`logging-es-ops-template`).
```

Comment:
@sosiouxme so it seems like the steps would be:

1. scale down everything (maintenance mode)
2. patch the current dc (maybe we can give them a patch command that will patch all DC for a cluster?)
3. patch the templates (again, maybe give a patch command)
4. create the new nodes from the patched templates
5. scale back up